from ca_seedwork.errors import DomainError


class UserNotFoundError(DomainError):
    pass
